src/modules/FioAddress/components/ConnectWallets.js

```diff
@@ -87,7 +87,7 @@ class ConnectWallets extends React.Component<Props, LocalState> {
       Actions.push(FIO_CONNECT_TO_WALLETS_CONFIRM, {
         fioAddressName,
         fioWallet,
-        walletsToConnect: Object.values(connectWalletsMap),
+        walletsToConnect: (Object.values(connectWalletsMap): any),
```

Comment:
Insert the line `const walletsToConnect: FioConnectionWalletItem[] = Object.keys(connectWalletsMap).map(key => connectWalletsMap[key])` above this block, and then you can just have `walletsToConnect` by itself on this line.